modules/telestion-api/src/main/java/de/wuespace/telestion/api/verticle/trait/DecodedMessage.java

```diff
@@ -2,11 +2,59 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import de.wuespace.telestion.api.message.JsonMessage;
+import io.vertx.core.Future;
 import io.vertx.core.eventbus.Message;
 import io.vertx.core.json.JsonObject;
 
+/**
+ * A wrapper for a decoded message from the {@link Message Vert.x message} to the {@link JsonMessage} type
+ * and basis {@link Message Vert.x message}
```

Comment:
To be completely honest: I don't quite understand what you intended on saying here. Please rephrase.

Reply:
Done in [`2cd29d6` (#484)](https://github.com/wuespace/telestion-core/pull/484/commits/2cd29d6f1426056b991dcbdcda1b3d2c3274fe9e)